src/js_sync.hpp

```diff
@@ -28,8 +28,10 @@
 #include "logger.hpp"
 
 #include "platform.hpp"
+#include "realm/object-store/shared_realm.hpp"
```

Comment:
Shouldn't this use angle brackets instead of quotes?

Reply:
It will follow the code standard -> yes!